### PR TITLE
Update to require sensu-plugin 2.7

### PR DESCRIPTION
**Pull request by jspaleta**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [X] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [X] RuboCop passes

- [X] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose
Update the gemspec to use sensu-plugin  2.7 which includes the logic to map sensu 2 events into sensu 1.5 events for handlers and mutators based on the sensu-plugin Handler and Mutator classes.

### CODE REVIEW

**Comment by jspaleta** [comment]
no idea why travis builds are failing. the errors are assocatied with kitchen.. so it seams like some sort of travis specific breakage.. not something in the commit.

**Comment by majormoses** [comment]
In the [logs](https://travis-ci.org/sensu-plugins/sensu-plugins-logs/jobs/431983839#L1548) it indicates that `net-telnet` version `0.2.0` dropped ruby support for < 2.3 in https://github.com/ruby/net-telnet/commit/4b5432a1d4f81e6c56bac6e72b0fe27bb8a6dfd8#diff-de0020d29d91153b0fe757ca69118a4e I am looking into what needs to be fixed, might be fastest/easiest to just drop those older versions support, let me get back to you on that one.

**Comment by majormoses** [comment]
Re-running tests as we should now be pinned properly if we pull in `specinfra` version `v2.76.2`

**Comment by majormoses** [comment]
@jspaleta I got one of the tests to pass now its just ruby 2.1 I am pretty sure that this can be temp worked around by adding this to the gemspec:
```
s.add_runtime_dependency 'net-ssh', '~> 4.0'
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [Unreleased]
 ### Added
-- updateed to use sensu-plugin 2.7 with 2.x to 1.x migration option
+- updated to use sensu-plugin 2.7 with 2.x to 1.x migration option
+### Changed
```
Comment (majormoses) [inline CHANGELOG.md]: This needs to be a `### Breaking Changes` header but I can fix that for you.

**Review by majormoses — APPROVED** [review]

**Review by majormoses — CHANGES_REQUESTED** [review]
We need to update https://github.com/sensu-plugins/sensu-plugins-logs/blob/9fc70a68103048893d18a2c1ba99c1408fa12298/.travis.yml#L12-L13 as well.

**Comment by jspaleta** [comment]
Okay made the change in supported ruby versions a breaking change, and bumped major version accordingly.  Let me know if there's anything else.

File: lib/sensu-plugins-logs/version.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 module SensuPluginsLogs
   module Version
-    MAJOR = 1
-    MINOR = 3
-    PATCH = 2
+    MAJOR = 2
```
Comment (majormoses) [inline lib/sensu-plugins-logs/version.rb:3]: its ok in this case but in general we do the actual versioning post acceptance.

**Review by majormoses — APPROVED** [review]

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-logs/versions/2.0.0

**Pull request merged**